src/liger_kernel/chunked_loss/fused_linear_preference.py

```diff
@@ -408,7 +408,7 @@ def _compute_loss(
         else:
             preference_loss, aux_outputs = preference_loss_outputs, []
 
-        loss = alpha * chosen_nll_loss - preference_loss
```

Comment:
Seems like this is correct for all losses except dpo_loss. I think the change the sign change should be here instead: https://github.com/linkedin/Liger-Kernel/blob/21bacccd107b40146e599e2c27af46d7d157f174/src/liger_kernel/chunked_loss/dpo_loss.py#L52

Reply:
Hi @shivam15s  Thanks for noticing this. What are your thoughts on negating each preference loss term to align with the formulas in the docstrings? This would allow us to maintain the base preference structure as `nll_loss + preference_loss` while making both terms represent losses to be minimized.

Reply:
Sure @austin362667 ! I think that might help with readability.
If you have some time, could you add the fix to this PR?

Reply:
Ya sure! I might need to open a new PR (base on this one) cc @winglian